.github/workflows/ci.yml

```diff
@@ -64,59 +75,57 @@ jobs:
-        pip install .[dev]
-    - name: Setup test config and CouchDB database server
-      run: |
-        python test/_helper/setup_testdb.py -u "admin" -p "$COUCHDB_ADMIN_PASSWORD"
-    - name: Test with coverage + unittest
-      run: |
-        coverage run --source=basyx -m unittest
-    - name: Report test coverage
-      if: ${{ always() }}
-      run: |
-        coverage report -m
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Collect schema files from aas-specs
+        run: |
+          mkdir -p ./test/adapter/schema
+          curl -sSL -o ./test/adapter/schema/aasJSONSchema.json https://raw.githubusercontent.com/admin-shell-io/aas-specs/${{ env.AAS_SPECS_RELEASE_TAG }}/schemas/json/aas.json
+          curl -sSL -o ./test/adapter/schema/aasXMLSchema.xsd https://raw.githubusercontent.com/admin-shell-io/aas-specs/${{ env.AAS_SPECS_RELEASE_TAG }}/schemas/xml/AAS.xsd
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[dev]
+      - name: Setup test config and CouchDB database server
+        run: |
+          python test/_helper/setup_testdb.py -u "admin" -p "$COUCHDB_ADMIN_PASSWORD"
+      - name: Test with coverage + unittest
+        run: |
+          coverage run --source=basyx -m unittest
+      - name: Report test coverage
+        if: ${{ always() }}
+        run: |
+          coverage report -m
 
   sdk-static-analysis:
     # This job runs static code analysis, namely pycodestyle and mypy
     runs-on: ubuntu-latest
+    needs: python-matrix
+    strategy:
+      matrix:
+        python-version: ${{fromJson(needs.python-matrix.outputs.versions)}}
     defaults:
       run:
         working-directory: ./sdk
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ env.X_PYTHON_MIN_VERSION }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ env.X_PYTHON_MIN_VERSION }}
-    - name: Install Python dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install .[dev]
-    - name: Check typing with MyPy
-      run: |
-        mypy basyx test
-    - name: Check code style with PyCodestyle
-      run: |
-        pycodestyle --count --max-line-length 120 basyx test
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[dev]
+      - name: Check typing with MyPy
+        run: |
+          mypy basyx test
+      - name: Check code style with PyCodestyle
+        run: |
+          pycodestyle --count --max-line-length 120 basyx test
```

Comment:
Could it be that the indentation broke here at some points?

Reply:
No, this was an automatic linting of PyCharm